.github/workflows/validate-stacks.yaml

```diff
@@ -163,17 +163,17 @@ jobs:
         with:
           go-version: "1.19"
 
-      - name: Install odo v3
-        uses: redhat-actions/openshift-tools-installer@2de9a80cf012ad0601021515481d433b91ef8fd5 # v1
-        with:
-          odo: "3.15.0"
+      - name: Install odo latest version
+        run: |
+          curl -L https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-nightly-builds/odo-linux-amd64 -o odo
+          sudo install -m 0755 odo /usr/local/bin/odo
+          mkdir -p $HOME/bin 
+          cp ./odo $HOME/bin/odo
+          export PATH=$PATH:$HOME/bin
```

Comment:
Is there a reason to place the binary in the home directory bin in addition to installing to `/usr/local/bin/odo`?

Reply:
I've added this following the instructions for the nightly builds. I think is a good point because I too not understand why. I've removed the commands and  brought back the temporary stack change (to see how the workflow behaves).

Reply:
Seems to be working https://github.com/devfile/registry/actions/runs/8848903793/job/24299796956?pr=372

I'm reverting again the temporary update of the stack.